### PR TITLE
Fix pylint issues for rawhide build

### PR DESCRIPTION
**Pull request by michalfabik**
Fix/disable issues from Pylint 2.4, see http://pylint.pycqa.org/en/latest/whatsnew/2.4.html

Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

File: src/pyfaf/actions/shell.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def any(obj):
             return session.query(obj).order_by(func.random()).first()
 
         try:
-            import IPython
+            import IPython # pylint: disable=import-outside-toplevel
```
Comment (ernestask) [inline src/pyfaf/actions/shell.py]: What if we just disabled it globally?

File: src/pyfaf/storage/jsontype.py [hunk 2]
```diff
@@ -18,5 +18,5 @@ def process_result_value(self, value, dialect):
     def process_literal_param(self, value, dialect):
         return value
 
-    def python_type(self):
+    def python_type(self): # pylint: disable=invalid-overridden-method
```
Comment (ernestask) [inline src/pyfaf/storage/jsontype.py]: Probably should rather just include a decorator? https://github.com/sqlalchemy/sqlalchemy/blob/8906617cd180f45219de2dd34dff8284f168fa9a/lib/sqlalchemy/sql/type_api.py#L392

**Comment by ernestask** [comment]
I would be for disabling warnings for non-toplevel imports in pylintrc, since sprinkling more disabler comments is stretching the definition of “fixing”.

File: src/pyfaf/actions/shell.py (hunk 1 above)
Comment (michalfabik) [reply src/pyfaf/actions/shell.py]: Disabled.

File: src/pyfaf/storage/jsontype.py (hunk 2 above)
Comment (michalfabik) [reply src/pyfaf/storage/jsontype.py]: Done.

**Comment by michalfabik** [comment]
Resolves #826

**Pull request merged**